### PR TITLE
Add use case to Readme.md

### PR DESCRIPTION
**Pull request by rsignell-usgs**
Adding a specific use case to Readme.md

### CODE REVIEW

**Comment by kwilcox** [comment]
Can you take out the `cache` folder specific stuff?  Since its parent folder is mounted it isn't actually doing anything

**Comment by rsignell-usgs** [comment]
done

**Comment by kwilcox** [comment]
There is one more you added

**Comment by kwilcox** [comment]
Poke

File: README.md [hunk 1]
```diff
@@ -31,7 +31,10 @@ $ docker run \
     -v /path/to/your/ssl.crt:/opt/tomcat/conf/ssl.crt \
     -v /path/to/your/ssl.key:/opt/tomcat/conf/ssl.key \
     -v /path/to/your/tomcat-users.xml:/opt/tomcat/conf/tomcat-users.xml \
+    -v /path/to/your/cache:/opt/tomcat/content/thredds/cache \
```
Comment (kwilcox) [inline README.md]: Remove this

**Comment by rsignell-usgs** [comment]
okay, removed.

**Pull request merged**